### PR TITLE
Selenium 4

### PR DESCRIPTION
**Pull request by karanshah-browserstack**

### CODE REVIEW

File: scripts/local.py [hunk 1]
```diff
@@ -19,26 +20,32 @@
 bs_local_args = { "key": BROWSERSTACK_ACCESS_KEY }
 
 # Starts the Local instance with the required arguments
+try:
+  bs_local.start(**bs_local_args)
+except Exception:
+  bs_local.stop()
+
 bs_local.start(**bs_local_args)
```
Comment (kamal-kaur04) [inline scripts/local.py:28]: this can be removed

File: scripts/local.py [hunk 2]
```diff
@@ -19,26 +20,32 @@
...
-  'realMobile': 'true',
-  'os_version': '14.0',
-  'name': 'BStack-[Python] Sample Test', # test name
-  'build': 'BStack Build Number 1', # CI/CD job or build name
-  'browserstack.local': 'true',
-  'browserstack.user': BROWSERSTACK_USERNAME,
-  'browserstack.key': BROWSERSTACK_ACCESS_KEY
+    "os" : "OS X",
+    "osVersion" : "Sierra",
+    "buildName" : "Final-Snippet-Test",
```
Comment (kamal-kaur04) [inline scripts/local.py:36]: Build Name and Session Name should remain as it is

**Review by kamal-kaur04 — CHANGES_REQUESTED** [review]

File: scripts/local.py (hunk 2 above)
Comment (karanshah-browserstack) [reply scripts/local.py:36]: It can be different

File: scripts/local.py (hunk 1 above)
Comment (karanshah-browserstack) [reply scripts/local.py:28]: Existing changes, can be done later

**Review by kamal-kaur04 — APPROVED** [review]

**Pull request merged**